src/lib/Microsoft.Health.Events/Telemetry/Metrics/EventMetrics.cs

```diff
@@ -21,16 +21,21 @@ public static class EventMetrics
         private static readonly string _operationDimension = DimensionNames.Operation;
         private static readonly string _reasonDimension = DimensionNames.Reason;
 
-        private static string _connectorOperation = ConnectorOperation.Unknown;
+        private static string _connectorOperation = Microsoft.Health.Common.Telemetry.ConnectorOperation.Unknown;
 
-        public static void SetConnectorOperation(string connectorOperation)
+        public static string ConnectorOperation
```

Comment:
nit: Just curious - any reason to change to property? 
Personally speaking, its not intuitive that properties are one time set so its easy to mistake this as an updateable one until we hit a runtime exception. But that's just my thinking. 
Or maybe consider adding a comment indicating this is a one time set property so that it shows in intellisense, etc.

Reply:
In commit 65a6c34, since I needed a `get` accessor for the operation to specify the operation as Normalization/FhirTransformation, I thought I'd convert `SetConnectorOperation` to a `set` accessor to pair the accessors up under one `ConnectorOperation` property.

I reverted commit 65a6c34 in commit 2a6520f (see other comment), so this change got reverted, but I'll keep your point in mind in the future, thanks!